esmvaltool/diag_scripts/radiation_budget/radiation_budget.py

```diff
@@ -0,0 +1,462 @@
+"""Radiation budget diagnostic script.
+
+PLACEHOLDER modified from AA docs: The aim of monitoring the energy
```

Comment:
How about extending it with: "This diagnostic analyses the radiation budget by separating top-of-atmosphere fluxes into clear-sky and cloud forcing components, and surface fluxes into downwelling and upwelling components. Model predictions are compared against three observational estimates, one of which (Stephens et al, 2012) includes uncertainty estimates. When the black error bars overlap the zero line, the model is consistent with observations according to Stephens et al." If this is too long for a doc string, then just the first sentence would be fine. The whole thing should go in the overview section of the readthedocs page though.

Reply:
The module doc strings were updated in https://github.com/ESMValGroup/ESMValTool/pull/2282/commits/6858cce6da9a9e415e122bdfb8bf0e30c53e9275. This change removed the placeholder summary. Would you like the summary reinstated and your new extension added, or shall I just add the extension to what is there currently?

Reply:
Ah yes, some of my comments were outdated because I started the review weeks ago. I didn't realise that GitHub withholds your comments until you complete the review.

No need to change the code, but I would add something like this text to the readthedocs overview which is a bit too brief as it stands.

Reply:
I've updated the radiation budget documentation in https://github.com/ESMValGroup/ESMValTool/pull/2282/commits/86a14475b3b81cd38f2256856425436d96c3d749.

Reply:
Perfect!

Reply:
I've also updated the recipe description in https://github.com/ESMValGroup/ESMValTool/pull/2282/commits/77796bd3ee76832017caace8805ca18072d7c4dc to match the documentation.